rest/route/aws.go

```diff
@@ -19,23 +26,32 @@ const (
 	messageTypeSubscriptionConfirmation = "SubscriptionConfirmation"
 	messageTypeNotification             = "Notification"
 	messageTypeUnsubscribeConfirmation  = "UnsubscribeConfirmation"
+	interruptionWarningType             = "EC2 Spot Instance Interruption Warning"
```

Comment:
Is there a list of these somewhere that we can link or?

Reply:
I couldn't find one for event types.
I've added a link for message types.